lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1945,6 +1942,27 @@ sub packages_diff {
     return join("\n", grep { !/(^@@|$ignore)/ } split(/\n/, $diff_packages));
 }
 
+=head2 done_hook_new_issue
+
+Called if job is done but no issue was carried over. If an environment
+variable corresponding to the found job result is found following the name
+pattern C<OPENQA_JOB_DONE_HOOK_$RESULT>, any executable specified in the
+variable as absolute path or executable name in C<$PATH> is called with the
+job ID as first and only parameter. For example for a job with result
+"failed", the corresponding environment variable would be
+C<OPENQA_JOB_DONE_HOOK_FAILED>. As alternative to an environment variable a
+corresponding config variable without the C<OPENQA_> prefix can be used in the
+format C<JOB_DONE_HOOK_$RESULT>. The corresponding environment value has
+precedence. The exit code of the externally called script is not evaluated and
+will have no effect.
+
+=cut
+sub done_hook_new_issue {
+    my ($self, $hook) = @_;
+    my $id = $self->id;
+    qx{$hook $id};
```

Comment:
I'd rather run this as a Minion job. It should be fairly simple to create a Minion task to run an arbitrary command and enqueue such a job here.

Reply:
this is a must. Don't run random scripts in a blocking fashion from within the webui - or you'll find yourself in front of 504 pages more often

Reply:
But don't we have enough prefork processes?

Reply:
But yeah, I was thinking of gru jobs as well. Just that I was not sure how to access the app->gru object properly here. I thought maybe it's necessary to to move all to https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

Reply:
enough prefork processes for what? We have tons of jobs finishing all the damn time

Reply:
You can get the "app" from everywhere via `OpenQA::App->singleton` and via that you can access gru and minion, e.g. to enqueue a job like `OpenQA::App->singleton->enqueue(run_script => [$path])`. (Of course the `run_script` task or something similar still needed to be created first. Just follow the patterns we're already using for other jobs.)

Reply:
solved in https://github.com/os-autoinst/openQA/pull/3603/commits/2eeb04dcd511de973265f9415524ca8cffe9ca00